docs/contributor/07-module-migration-guide.md

```diff
@@ -0,0 +1,170 @@
+   - ModuleTemplate `kyma-system/telemetry-1.34.0-experimental` (only in DEV and STAGE)
+   - ModuleTemplate `kyma-system/telemetry-1.35.0-rc1` (only in DEV)
+
+   As the new module metadata takes precedence, the reconciliation of the module already happens based on the new metadata. Since all versions and channel mappings are the same, no update is performed and all modules stay in the same state as before.
+
+   The functionality can further be verified by enabling the module in a test SKR which will install it from scratch using the new metadata.
+
+4. [OPTIONAL] Roll back the new module metadata.
+
+   In case of failure, the setup can be reverted to the old approach.
+
+   To do so, a PR can be opened to `/kyma/kyma-modules` reverting the submission from 2. ArgoCD then undeploys the new module metadata and KLM falls back to the old module metadata.
+
+   > [!Note]
+   > After rollback, you can still use the old submission pipeline to submit new versions of the module while you're working on a fix.
+
+5. Submit a version upgrade using the old format.
+
+   To avoid version downgrades, prepare for failure recovery and submit a version update using the old metadata.
+
+   - `/modules/telemetry/regular/module-config.yaml` pointing to `1.34.0` (before `1.32.0`)
+
+   Since the new metadata exists, Lifecycle Manager continues to use it. The old metadata is ignored but remains available for rollback.
+
+6. Submit the updated channel mapping with the NEW approach.
+
+   After preparing the old metadata to rollback in case of failure, you can continue with the actual version update using the new metadata.
+
+   Following the example, submit the `modules/telemetry/module-releases.yaml` file:
+
+   ```yaml
+   channels:
+     - channel: regular
+       version: 1.34.0 # <= this version is bumped
+     - channel: fast
+       version: 1.34.0
+     - channel: experimental
+       version: 1.34.0-experimental
+     - channel: dev
+       version: 1.35.0-rc1
+   ```
+
+   Once you submit the mapping, the resources in `/kyma/kyma-modules` are the same as the resources from step 2, except the `regular` channel pointing to version `1.34.0` in all ModuleReleaseMeta CRs for different landscapes.
+
+7. Verify if the module is updated in KCP.
+
+   ArgoCD picks up this change and deploys the new ModuleReleaseMeta to the different landscapes. Lifecycle Manager is now picking up the version change and updating all modules using the `regular` channel to version `1.34.0`.
+
+8. [OPTIONAL] Roll back the new metadata.
+
+   In case of failure, you can revert the setup to the old approach.
+
+   To do so, a PR can be opened to `/kyma/kyma-modules` reverting the submissions from steps 2 and 6. It is important to revert completely removing the entire new metadata from KCP so that KLM falls back to the old module metadata.
+
+   > [!Note]
+   > After rollback, you can still use the old submission pipeline to submit new versions of the module while you're working on a fix.
```

Comment:
Note doesn't seem to be rendered correctly.

Reply:
Once merged, it will apparently 🤷  Will check and do a post merge fix is needed.